exercises/concept/cars-assemble/.docs/introduction.md

```diff
@@ -0,0 +1,185 @@
+# Introduction
+
+## Numbers
+
+There are two different types of numbers in Rust:
+
+- Integers: numbers with no digits behind the decimal separator (whole numbers). Examples are `-6`, `0`, `1`, `25`, `976` and `500000`.
+- Floating-point numbers: numbers with zero or more digits behind the decimal separator. Examples are `-2.4`, `0.1`, `3.14`, `16.984025` and `1024.0`.
+
+The two default numeric types in Rust are `i32` and `f64`. An `i32` is a 32-bit integer and a `f64` is a 64-bit floating-point number.
```

Comment:
They aren't used in the exercise, but it might be appropriate to include a note about the other number types? Something like:

> There are other number types with different precision, like `f32` and `i16`. Unsigned (always positive) integers start with a `u` instead of `i`. For example, a byte is expressed as `u8`. Another number type you'll come across is `usize`, its precision depends on your computer (usually `64` bits) and is used for lengths and indexes.